### PR TITLE
Add Oak Containers environment variable support

### PR DESCRIPTION
**Pull request by conradgrobler**
So far we have ignored environment variables in the container config, but we might need it for some containers.

This also changes some of the orchestrator's logging to make the output more readable and relevant.

### CODE REVIEW

File: oak_containers_orchestrator/src/container_runtime.rs [hunk 1]
```diff
@@ -145,11 +151,21 @@ pub async fn run(container_bundle: &[u8]) -> Result<(), anyhow::Error> {
...
+        // Run the trusted app in a chroot environment of the container.
         std::os::unix::fs::chroot(container_rootfs_path.clone())?;
-        // Set the specified working directory
-        std::env::set_current_dir(oci_filesystem_bundle_config.process.cwd.clone())?;
+        let cwd = oci_filesystem_bundle_config.process.cwd.clone();
+        if cwd.has_root() {
+            std::env::set_current_dir(cwd)?;
+        }
+        for variable in oci_filesystem_bundle_config.process.env.clone() {
+            if let Some((key, value)) = variable.split_once('=') {
```
Comment (mariaschett) [inline oak_containers_orchestrator/src/container_runtime.rs:165]: Do you assume the given enviroment variables to be exactly in this format: VAR=VAL ?

What about e.g., VAR = VAL or VAR= VAL? Will this be a problem? 

Is there some library support for parsing enviroment variables or can you add documentation?

I think I remember spending a couple on hours on tracking down that " FOO" is not "FOO".

File: oak_containers_orchestrator/src/container_runtime.rs [hunk 2]
```diff
@@ -136,6 +141,7 @@ pub async fn run(container_bundle: &[u8]) -> Result<(), anyhow::Error> {
     };
     rmount_dir(std::path::PathBuf::from("/proc"), container_proc_path).await?;
 
+    log::info!("Command: {:?}", oci_filesystem_bundle_config.process.args);
```
Comment (mariaschett) [inline oak_containers_orchestrator/src/container_runtime.rs:144]: maybe: `Startup command` ?

File: oak_containers_orchestrator/src/container_runtime.rs [hunk 3]
```diff
@@ -94,6 +97,8 @@ pub async fn run(container_bundle: &[u8]) -> Result<(), anyhow::Error> {
         serde_json::from_str(&oci_filesystem_bundle_config_file)?
     };
 
+    log::info!("Setting up container");
```
Comment (mariaschett) [inline oak_containers_orchestrator/src/container_runtime.rs:100]: maybe, if easy and helpful. Is there a way we can identify the container here, e.g., print the container name or so?

e.g. `log::info!("Setting up container from {}", CONTAINER_DIR)`.

**Review by mariaschett — APPROVED** [review]

File: oak_containers_orchestrator/src/container_runtime.rs (hunk 2 above)
Comment (conradgrobler) [reply oak_containers_orchestrator/src/container_runtime.rs]: Done.

File: oak_containers_orchestrator/src/container_runtime.rs (hunk 1 above)
Comment (conradgrobler) [reply oak_containers_orchestrator/src/container_runtime.rs]: These come from the container image. If there was a space, these would be included when running the container in e.g. Docker as well, and I don't think we should change the behaviour.

File: oak_containers_orchestrator/src/container_runtime.rs (hunk 3 above)
Comment (conradgrobler) [reply oak_containers_orchestrator/src/container_runtime.rs:92]: We don't have identifying information like a name in the Runtime bundle. The root directory is also named the same for all container images, so I don't think that provides useful information.

**Review by conradgrobler — COMMENTED** [review]
Thanks.

**Pull request merged**